webpack/client/production.ts

```diff
@@ -37,7 +37,7 @@ export default (): Configuration => {
           test: /.*\.(gif|png|jpe?g)$/i,
           type: 'asset/resource',
           generator: {
-            filename: 'images/[name].[hash][ext]'
+            filename: 'images/[name].[contenthash][ext]'
```

Comment:
why this change? whats the difference betwee `hash` key and the `contenthash` key

Reply:
Here's the [full documentation](https://webpack.js.org/configuration/output/#template-strings) of webpack's options. `hash` is a deprecated option, replaced by `fullhash`. In [another place](https://webpack.js.org/concepts/under-the-hood/#output) in the docs, they say that the most common option is `contenthash`. Contenthash should refer to the hash from the content of the given generated file; so in theory, should be the most appropriate for caching purposes.